sos/report/plugins/ovirt.py

```diff
@@ -91,7 +91,37 @@ def setup(self):
             # Copy all engine tunables and domain information
             "engine-config --all",
             # clearer diff from factory defaults (only on ovirt>=4.2.8)
-            "engine-config -d"
+            "engine-config -d",
+            # oVirt CA cert.
+            "openssl x509 -in /etc/pki/ovirt-engine/ca.pem "
+            "-text -noout",
+            # oVirt Apache CA cert.
+            "openssl x509 -in /etc/pki/ovirt-engine/apache-ca.pem "
+            "-text -noout",
+            # oVirt Engine cert.
+            "openssl x509 -in /etc/pki/ovirt-engine/certs/engine.cer "
+            "-text -noout",
+            # oVirt Apache cert.
+            "openssl x509 -in /etc/pki/ovirt-engine/certs/apache.cer "
+            "-text -noout",
+            # oVirt WebSocket cert.
+            "openssl x509 -in /etc/pki/ovirt-engine/certs/websocket-proxy.cer "
+            "-text -noout",
+            # oVirt JBoss cert.
+            "openssl x509 -in /etc/pki/ovirt-engine/certs/jboss.cer "
+            "-text -noout",
+            # oVirt ImageIO Proxy cert.
+            "openssl x509 -in /etc/pki/ovirt-engine/certs/imageio-proxy.cer "
+            "-text -noout",
+            # oVirt OVN provider cert.
+            "openssl x509 -in /etc/pki/ovirt-engine/certs/ovirt-provider-"
+            "ovn.cer -text -noout",
```

Comment:
Since the vast majority of the added commands are the same, just targeting different certs, I'd suggest something like this:

```python
certs = [
    '/etc/pki/ovirt-engine/ca.pem',
    '/etc/pki/ovirt-engine/apache-ca.pem'
# etc...
]
self.add_cmd_output([
    "openssl x509 -in %s -text -noout" % c for c in certs
])
```